src/classes/TaskItemController.cls

```diff
@@ -0,0 +1,31 @@
+public with sharing class TaskItemController {
+
+	@AuraEnabled
+	public static List<Task__c> getTaskItems()
+	{
+		return [SELECT Id, Name, Status__c, Description__c, Contact__c, Contact__r.Name, Contact__r.Total_Points_Awared__c, RecordTypeId, RecordType.Name FROM Task__c];
+	}
+
+	@AuraEnabled
+	public static Contact getTaskContactApex(Task__c task)
```

Comment:
You should load any contact-related information in the getTaskItems method.

Reply:
Done